main/res/values/strings.xml

```diff
@@ -1590,5 +1596,8 @@
     <string name="init_low_disk_space_message">The geocache data directory is running low on storage space. This can cause malfunctions of c:geo. Please free up some space.</string>
     <string name="init_confirm_debug">c:geo is running in debug mode!</string>
     <string name="list_confirm_debug_message">The debug mode causes instabilities! Do you want to deactivate the debug mode?</string>
+    <string name="resault">Resault:</string>
+    <string name="equation_hint">a^2 + b^2</string>
```

Comment:
We could use the hint string here to show all the possible operators, like `a+b/c*d^2`